x-pack/plugins/observability/public/pages/home/index.tsx

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButton,
+  EuiFlexGrid,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiIcon,
+  EuiImage,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import { ISection, Section } from './section';
+import { CoreStart } from '../../../../../../src/core/public';
+
+const appsSection: ISection[] = [
+  {
+    id: 'logs',
+    title: 'Logs',
+    icon: 'logoLogging',
+    description:
+      'The Elastic Stack (sometimes known as the ELK Stack) is the most popular open source logging platform.',
+  },
+  {
+    id: 'apm',
+    title: 'APM',
+    icon: 'logoAPM',
+    description:
+      'See exactly where your application is spending time so you can quickly fix issues and feel good about the code you push.',
+  },
+  {
+    id: 'metrics',
+    title: 'Metrics',
+    icon: 'logoMetrics',
+    description:
+      'Already using the Elastic Stack for logs? Add metrics in just a few steps and correlate metrics and logs in one place.',
+  },
+  {
+    id: 'uptime',
+    title: 'Uptime',
+    icon: 'logoUptime',
+    description:
+      'React to availability issues across your apps and services before they affect users.',
+  },
+];
+
+const tryItOutItemsSection: ISection[] = [
+  {
+    id: 'demo',
+    title: 'Demo Playground',
+    icon: 'play',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed sit amet magna at neque dictum.',
+    href: 'https://demo.elastic.co/',
+  },
+  {
+    id: 'sampleData',
+    title: 'Add sample data',
+    icon: 'documents',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed sit amet magna at neque dictum.',
+    href: '/app/home#/tutorial_directory/sampleData',
```

Comment:
I think this and the Get started link needs some sort of dynamic link because it does not work when you have basepath enabled.